File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program31.cpp
///////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept number from user and display below pattern.
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Ans
{
	private:
		int iNo;
		int iCnt;
		char ch;

	public:
		Ans(int iNo,int iCnt=0)
		{
			this->iNo = iNo;
			this->ch = 'A';
		}

		void Display()
		{
			for(iCnt=1;iCnt<=iNo;iCnt++)
			{
				cout<<ch<<"\t";
				ch++;
			}

			cout<<"\n";
		}
};

int main()
{
	int iValue=0;

	cout<<"Enter number of elements: ";
	cin>>iValue;
	
	Ans *aObj = new Ans(iValue);
	aObj->Display();
}

/*
OUTPUT:

ENter number of elements: 5
A	B	C	D	E	

*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program31.java
///////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept number from user and display below pattern.
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{
	public int iValue;
	public char ch;
	public int iCnt;

	public Solution(int iValue)
	{
		this.iValue = iValue;
		this.ch = 'A';
		this.iCnt=0;
	}

	public void Display()
	{
		for(iCnt=1;iCnt<=iValue;iCnt++)
		{
			System.out.print(ch + "\t");
			ch++;
		}

		System.out.println();
	}
}

class Program1
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		System.out.println("Enter number of elements:");
		int iNo = sObj.nextInt();

		Solution obj = new Solution(iNo);
		obj.Display();
	}
}

/*
OUTPUT:

Enter number of elements:
5
A	B	C	D	E

*/
